### ISSUE TITLE
Table column with kebab action doesn't have a11y label

### ISSUE THREAD

**Issue by matthewcarleton** [patternfly/patternfly#3181]
When a screen reader reaches the 6th column in this table it announces it as column 6 of 6 but doesn't provide any other information. It would be helpful to add a label for a screen reader here. It could be a better overall experience to label this column as actions for better cognitive a11y support as well.
<img width="795" alt="Screen Shot 2020-06-16 at 5 06 31 PM" src="https://user-images.githubusercontent.com/287880/84822823-bcf04c80-aff3-11ea-8b08-c2f6463897d5.png">

**Comment by mcoker** [comment]
cc @jessiehuff

**Comment by jgiardino** [comment]
+1

@jessiehuff and I were chatting about this one, and a suggestion to provide a table header that includes a visually hidden but screen reader accessible string. Is this what you were thinking, or is your suggestion to have visible "Actions" label?

I'm also curious what the suggestion should be if a consumer wanted to have a more prominent action provided as a button that is separate from and adjacent to the kebab. Would these elements be inside two separate table cells? Would they share the same table header?

**Comment by matthewcarleton** [comment]
> +1
> 
> @jessiehuff and I were chatting about this one, and a suggestion to provide a table header that includes a visually hidden but screen reader accessible string. Is this what you were thinking, or is your suggestion to have visible "Actions" label?
> 
ya I think it's really a question of support. Are we assuming all sighted users will easily understand this or would adding an "Actions" label just make it that much easier for a wider group of users to understand. I can see this being a bit of a design challenge but worthy of exploration.

> I'm also curious what the suggestion should be if a consumer wanted to have a more prominent action provided as a button that is separate from and adjacent to the kebab. Would these elements be inside two separate table cells? Would they share the same table header?

Ya again for a screen reader to have a column announced as "Actions" and then have everything action related in there makes sense to me. I'm not sure if we are improving their experience by having a separate header that they have to listen to. Visually this would get more challenging too - space is always an issue.

**Comment by jessiehuff** [comment]
I agree that a label could add to the context of the table. It does sound like a design question of whether it's obvious enough that it's an actions column and should just include an accessible label or if, as mentioned, having an "Actions" label would also improve the sighted user experience. What do you think @mcarrano? Do you think it would make sense to add a visible label or just an accessible label only available to screen reader users?

**Comment by mcarrano** [comment]
I agree with what others have said above that adding a column header labeled "Actions" is both unnecessary and problematic.

**Comment by matthewcarleton** [comment]
> I agree with what others have said above that adding a column header labeled "Actions" is both unnecessary and problematic.

I don't feel as strongly about the text label but I often wonder how we determine adding value for sighted users with a cognitive disability.
